### PR TITLE
Add some features

### PR DESCRIPTION
**Pull request by biankaiwen111**
1. can read .json one by one automatically
2.change __init__ in Course class

### CODE REVIEW

File: ReadCourseData.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+		self.courses = list()
+
+class Lab:
+	def __init__(self, *args):
+		self.UID, self.days, self.startTime,\
+		self.endTime, self.startDate, self.endDate,\
+		self.instructor, self.location = args
+
+def main():
+	###Don't forget to put all the .json file in the current dir
```
Comment (yifeili98) [inline ReadCourseData.py:61]: Consider using more functions to reduce cognitive complexity in main()

File: ReadCourseData.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
+import os, json
+import os,inspect
```
Comment (yifeili98) [inline ReadCourseData.py:2]: nit: os is imported twice

**Comment by yifeili98** [comment]
Consider to use semantic commit style (https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716)

File: ReadCourseData.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+				if interesting_name not in temp_dept.courses:
+					temp_dept.courses.append(interesting_name)
+				total_course += 1
+			print('department', department, "has", dep_course_num, "sections.")
+			dlist.append(temp_dept)
+
+		print('total course:', total_course)
+		print('loaded course:', len(clist))
+
+
```
Comment (yifeili98) [inline ReadCourseData.py:111]: nit: redundant empty lines

File: ReadCourseData.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+def main():
+	###Don't forget to put all the .json file in the current dir
+	current_path = os.path.abspath('.')
+	current_path = current_path.replace('\\', '/')
+	print(current_path)
+	allfiles= os.listdir(current_path)
+	print(allfiles)
+	#file_abs_path = os.path.abspath(current_path) + "\\"
+	
+
```
Comment (yifeili98) [inline ReadCourseData.py:69]: nit: redundant empty lines

File: ReadCourseData.py [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+	username = input("Enter your username:")
+	password = input("Password:")
+	client = MongoClient("mongodb+srv://" + username +":" + password + "@fhdatimedb-jjsjm.mongodb.net/test?retryWrites=true&w=majority")
```
Comment (yifeili98) [inline ReadCourseData.py:72]: nit: maybe consider to use a .env file

**Review by yifeili98 — CHANGES_REQUESTED** [review]

**Comment by kis87988** [comment]
Please do `git pull origin master` and then fix all errors

**Review by hk-mp5a3 — COMMENTED** [review]
nit: do not use "print" in production. using log will be a better practice

**Pull request closed without merging**